backend/internal/storage/postgres/assignment_templates.go

```diff
@@ -50,3 +52,15 @@ func (db *DB) GetAssignmentTemplateByID(ctx context.Context, templateID int64) (
 
 	return assignmentTemplate, nil
 }
+
+func (db *DB) GetAssignmentDueDateByRepoName(ctx context.Context, repoName string) (*time.Time, error){
```

Comment:
See comment above about generalizing this return Assignment object and doing field extraction in service layer.

Reply:
✅